src/components/ShowList.jsx

```diff
@@ -1,11 +1,18 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { useCollection } from 'react-firebase-hooks/firestore';
 import db from '../lib/firebase';
 import { useHistory } from 'react-router-dom';
 import Item from './Item';
 import './ShowList.css';
+import { doc } from 'prettier';
+import { library } from '@fortawesome/fontawesome-svg-core';
+import { fas } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+library.add(fas);
```

Comment:
Wow that's a pretty Ideas, I think we should call it globally so that it can be reused in the other components

Reply:
This is a great idea, moving these lines to `index.js` would do the trick. 👍 

```js
import { library } from '@fortawesome/fontawesome-svg-core';
import { fas } from '@fortawesome/free-solid-svg-icons';
library.add(fas);
```

Reply:
These are amazing suggestions. I would also suggest lifting this up in the higher component for reusability reasons. 

I have one question about the FontAwesome Library. Did you install the whole library? Have you thought about using CDN(Content Delivery Networks) for the icon implementation? CDN (which in short is an internet server)  can help in multiple ways such as: `faster asset loading`, `scaling/distributing load`, `more reliable` and `accessible`. It also does not increase the overall size of the application since we do not install the whole package in our app. 

You can read more about it [here](https://cdn.fontawesome.com/help). 
For the installation instructions, check out [this link](https://cdn.fontawesome.com/)

Reply:
Thank you for the great feedback, we didn't consider the consequence of downloading the library or using the cdn.

Reply:
Yep, implementing the icons using CDN is pretty easy. There is an instruction in the article I shared. Consider using it because we will not be using a lot of icons from the package and it will take less memory, as well as will increase the performance 🙂